rides/models.py

```diff
@@ -11,12 +12,16 @@ class User(db.Model):
     firstname = db.Column(db.String, nullable=False)
     lastname = db.Column(db.String, nullable=False)
     picture = db.Column(db.String, nullable=False)
+    slack = db.Column(db.String, nullable=False) # ADD TO MYSQL
+    email = db.Column(db.String, nullable=False) # ADD TO MYSQL
 
-    def __init__(self, id, firstname, lastname, picture):
-        self.id = id
+    def __init__(self, uid, firstname, lastname, picture, slack, email):
+        self.id = uid
```

Comment:
Why are you changing this to `uid`?

Reply:
Pylint was concerned over writing some other id variable, but only for that model.